makerom/cia.c

```diff
@@ -184,6 +184,12 @@ int GetSettingsFromUsrset(cia_settings *ciaset, user_settings *usrset)
 	
 	if(usrset->cia.randomTitleKey)
 		rndset(ciaset->common.titleKey,AES_128_KEY_SIZE);
+	else if(usrset->cia.titleKey){
```

Comment:
Can you please change the indentation from spaces to tabs to match the existing code?